server/controllers/stock/requisition/requisition.js

```diff
@@ -316,5 +320,21 @@ exports.deleteRequisition = async (req, res, next) => {
   }
 };
 
+/**
+ * @function updateStatus
+ * @description updates the status of a requisition based
+ * @param {string} uuid - The uuid of the requisition
+ */
+async function updateStatus(uuid) {
+  // First get the requisition details
+  const balance = await getDetailsBalance(uuid);
+  if (balance.items.length === 0) {
+    return db.exec('UPDATE stock_requisition SET status_id = ? WHERE uuid = ?', [
```

Comment:
Since we are in an `async` context, we can avoid some of this complexity with return statements.  I propose you just do:

```js
const balance = await getDetailsBalance(uuid);
  if (balance.items.length === 0) {
    await db.exec('UPDATE stock_requisition SET status_id = ? WHERE uuid = ?', [
  REQUISITION_STATUS_COMPLETE, db.bid(uuid)]);
  }
````

Reply:
Done.  Thanks for the reminder!